database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
wasn't going to use a foreign key for that column was going to have it be an implicit association...but I added one 👍  and also took off `user_name` column